4-storage/postgres-storage/sql/migrate_0.7.0_to_0.7.1.sql

```diff
@@ -0,0 +1,26 @@
+-- Copyright (c) 2018 Snowplow Analytics Ltd. All rights reserved.
+--
+-- This program is licensed to you under the Apache License Version 2.0,
+-- and you may not use this file except in compliance with the Apache License Version 2.0.
+-- You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the Apache License Version 2.0 is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+--
+-- Version:     Ports version 0.7.0 to version 0.7.1
+-- URL:         -
+--
+-- Authors:     Mike Robins
+-- Copyright:   Copyright (c) 2018 Snowplow Analytics Ltd
+-- License:     Apache License Version 2.0
+
+BEGIN TRANSACTION;
```

Comment:
I actually don't think transaction works for `ALTER COLUMN`

Reply:
I tried this in 10.5 and it seemed to work. Should I try this on other versions?

Reply:
I just mean that it probably won't be a transaction, e.g. if `COMMENT` will fail (let's imagine) `ALTER COLUMN` won't be rolled back.

Reply:
I could be incorrect but I though that [Postgres DDLs](https://wiki.postgresql.org/wiki/Transactional_DDL_in_PostgreSQL%3a_A_Competitive_Analysis) are transactional so this would be rolled back.

Reply:
Okay, I'm actually happy to be wrong here.